bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/chart/ChartServlet.java

```diff
@@ -135,6 +132,16 @@ protected void deactivate() {
         httpService.unregister(SERVLET_NAME);
     }
 
+    protected void modified(Map<String, Object> config) {
```

Comment:
Yes, I read it the same way and thus I think the icon servlet xml is incorrect, that's what I mentioned.